faq: Add outstanding questions (#72)
radicle-dev/radicle-docs: docs/understanding-radicle/faq.md

## Can I change my passphrase?
Not yet — so make sure to keep it in a safe place!

## Why do I have to enter my passphrase everytime?

> Interactions through Git and the remote helper are adhoc and don't have the

Reviewer: ```suggestion
Interactions through Git and the remote helper are ad-hoc and don't have the
```

benefit of a long-running daemon - i.e. Upstream client. That means for now the

> passphrase as to be provided every time you interact with Radicle outside of

Reviewer: ```suggestion
passphrase has to be provided every time you interact with Radicle outside of
```

If you are still running into problems, please submit a request in our [#support
channel][sc].

## Why am I only connected to one peer?

> By default the Upstream client is conecting to a seed node operated by Radicle.

Reviewer: ```suggestion
By default, the Upstream client is conecting to a seed node operated by Radicle.
```

While we support [epidemic broadcast][eb] to find and connect to other peers, we

> have support [hole punching][hp] just yet, which will prevent a stable conenction

Reviewer: ```suggestion
don't have support [hole punching][hp] just yet, which will prevent a stable conenction
```

To join our Matrix chat with an account from another Matrix server, you can use
this [direct invite link][mc] to join #general.

## Can I run Radicle as a daemon?

> While technically possible we haven't bundled it yet in a convenient package

Reviewer: ```suggestion
While this is technically possible, we haven't bundled it yet in a convenient package
```

> for anyone to run in the background. We are working hard to change that and

Reviewer: ```suggestion
for anyone to run in the background. We are working hard to change that so we can
```

> help people operate Radicle peers in many different ways. Should you be keen to

Reviewer: ```suggestion
help people operate Radicle nodes in many different ways. Should you be keen to
```

> have it as a daemon right, check out how the [seed][si[ is implemented and try to run

Reviewer: Is this what you wanted this section to be?
```suggestion
have it as a daemon right now, check out how the [seed][si] is implemented, and try to run one on
```

> have it as a daemon right, check out how the [seed][si[ is implemented and try to run
> your own.

Author: ```suggestion
have it as a daemon right now, check out how the [seed][si[ is implemented, and
try to run your own.
```
